example/docker-compose/readme.md

```diff
@@ -30,44 +30,46 @@ Tempo can be run with local storage, S3, Azure, or GCS backends. See below for e
 In this example all data is stored locally in the example-data/tempo folder.
 
 1. First start up the local stack. 
-```
+```console
 docker-compose up -d
 ```
 
 At this point, the following containers should be spun up -
 
 ```console
-$ docker-compose -f docker-compose.local.yaml ps
```

Comment:
I think we need this because `docker-compose ps` would look up the services in the file `docker-compose.yaml` by default.

Reply:
honestly it might be better to just remove the `ps` command to make this file leaner

Reply:
This example doesn't need the `-f` flag because it is using the default `docker-compose.yaml`. But maybe this is confusing?

Reply:
We had `docker-compose.local.yaml`: https://github.com/grafana/tempo/blob/v0.2.0/example/docker-compose/docker-compose.local.yaml

But we no longer have that file... so I think it's alright to remove the flag.